src/app/components/filter/filter.component.ts

```diff
@@ -0,0 +1,28 @@
+import {
+    Component,
+    EventEmitter,
+    Output
+} from '@angular/core';
+
+/**
+ * @author mkunze
+ * @description This component replaces <terra-filter>
```

Comment:
Was ist wenn `<terra-filter>` mal nicht mehr da ist? 😝 
```suggestion
 * @description This component provides the default template and functionalities to display form fields which are supposed to set filters
```